common/src/cams/events.test.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  Consolidation,
+  isJointAdministrationChildCase,
+  isJointAdministrationLeadCase,
+} from './events';
+import { MockData } from './test-utilities/mock-data';
+
+const jointAdministrationLeadCase: Consolidation[] = [
+  MockData.getConsolidationReference({
+    override: { documentType: 'CONSOLIDATION_FROM', consolidationType: 'administrative' },
+  }),
+];
+const jointAdministrationChildCase: Consolidation[] = [
+  MockData.getConsolidationReference({
+    override: { documentType: 'CONSOLIDATION_TO', consolidationType: 'administrative' },
+  }),
+];
+const substantiveConsolidationLeadCase: Consolidation[] = [
+  MockData.getConsolidationReference({
+    override: { documentType: 'CONSOLIDATION_FROM', consolidationType: 'substantive' },
+  }),
+];
+const substantiveConsolidationChildCase: Consolidation[] = [
+  MockData.getConsolidationReference({
+    override: { documentType: 'CONSOLIDATION_TO', consolidationType: 'substantive' },
+  }),
+];
+
+describe('Event domain helper functions', () => {
+  describe('isJointAdministrationLeadCase', () => {
+    test('should return true', () => {
+      expect(isJointAdministrationLeadCase(jointAdministrationLeadCase)).toBeTruthy();
+    });
+    test('should return false for all other cases', () => {
+      expect(isJointAdministrationLeadCase([])).toBeFalsy();
+      expect(isJointAdministrationLeadCase(undefined)).toBeFalsy();
+      expect(isJointAdministrationLeadCase(jointAdministrationChildCase)).toBeFalsy();
+      expect(isJointAdministrationLeadCase(substantiveConsolidationLeadCase)).toBeFalsy();
```

Comment:
Should we also be testing `substantiveConsolidationChildCase` and `nonConsolidatedCase`?

Reply:
Looks good per conversation.